.github/CODEOWNERS

```diff
@@ -16,7 +16,7 @@
 /eng/common/                                                                      @dotnet-maestro-bot
 /eng/Versions.props                                                               @dotnet-maestro-bot @dotnet/aspnet-build @wtgodbe
 /eng/Version.Details.xml                                                          @dotnet-maestro-bot @dotnet/aspnet-build @wtgodbe
-/eng/SourceBuild*                                                                 @dotnet/source-build-internal
+/eng/SourceBuild*                                                                 @dotnet/source-build
```

Comment:
Same as https://github.com/dotnet/aspire/pull/6210#discussion_r1794012004, do we have write access?